From 0eaad2d7475d41b9bb7afcecbe99aaafcc8a2761 Mon Sep 17 00:00:00 2001
From: Paul Bolle <pebolle@tiscali.nl>
Date: Mon, 15 Aug 2011 02:02:26 +0200
Subject: doc: fix broken references

There are numerous broken references to Documentation files (in other
Documentation files, in comments, etc.). These broken references are
caused by typo's in the references, and by renames or removals of the
Documentation files. Some broken references are simply odd.

Fix these broken references, sometimes by dropping the irrelevant text
they were part of.

Signed-off-by: Paul Bolle <pebolle@tiscali.nl>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
(cherry picked from commit 395cf9691d72173d8cdaa613c5f0255f993af94b)

Conflicts:

	Documentation/PCI/pci.txt
	Documentation/blackfin/bfin-gpio-notes.txt
	Documentation/block/biodoc.txt
	Documentation/bus-virt-phys-mapping.txt
	Documentation/cdrom/packet-writing.txt
	Documentation/development-process/4.Coding
	Documentation/devicetree/bindings/gpio/led.txt
	Documentation/filesystems/caching/object.txt
	Documentation/filesystems/locks.txt
	Documentation/filesystems/nfs/idmapper.txt
	Documentation/filesystems/pohmelfs/design_notes.txt
	Documentation/filesystems/proc.txt
	Documentation/filesystems/vfs.txt
	Documentation/frv/booting.txt
	Documentation/input/input.txt
	Documentation/kernel-docs.txt
	Documentation/kernel-parameters.txt
	Documentation/laptops/thinkpad-acpi.txt
	Documentation/media-framework.txt
	Documentation/memory-barriers.txt
	Documentation/networking/scaling.txt
	Documentation/power/userland-swsusp.txt
	Documentation/rfkill.txt
	Documentation/scsi/aic7xxx_old.txt
	Documentation/scsi/scsi_mid_low_api.txt
	Documentation/security/keys-trusted-encrypted.txt
	Documentation/sound/oss/PAS16
	Documentation/spi/pxa2xx
	Documentation/timers/highres.txt
	Documentation/usb/dma.txt
	Documentation/virtual/lguest/lguest.c
	Documentation/vm/numa
	Documentation/vm/slub.txt
	arch/alpha/kernel/srm_env.c
	arch/arm/Kconfig
	arch/arm/include/asm/io.h
	arch/arm/mach-pxa/xcep.c
	arch/ia64/hp/common/sba_iommu.c
	arch/m68k/q40/README
	arch/microblaze/include/asm/dma-mapping.h
	arch/mips/include/asm/lasat/lasat.h
	arch/mn10300/Kconfig
	arch/mn10300/kernel/irq.c
	arch/openrisc/Kconfig
	arch/openrisc/include/asm/dma-mapping.h
	arch/parisc/include/asm/dma-mapping.h
	arch/parisc/kernel/pci-dma.c
	arch/powerpc/include/asm/qe.h
	arch/powerpc/sysdev/qe_lib/qe.c
	arch/unicore32/include/asm/io.h
	arch/x86/Kconfig
	arch/x86/Kconfig.debug
	arch/x86/boot/header.S
	arch/x86/include/asm/dma-mapping.h
	arch/x86/kernel/amd_gart_64.c
	arch/x86/kernel/apm_32.c
	arch/x86/kernel/pci-dma.c
	drivers/char/apm-emulation.c
	drivers/input/misc/rotary_encoder.c
	drivers/leds/Kconfig
	drivers/media/dvb/dvb-usb/af9005-remote.c
	drivers/media/dvb/dvb-usb/af9005.c
	drivers/media/dvb/frontends/dib3000.h
	drivers/media/dvb/frontends/dib3000mb.c
	drivers/mtd/Kconfig
	drivers/net/Kconfig
	drivers/net/can/sja1000/sja1000_of_platform.c
	drivers/net/tulip/21142.c
	drivers/net/tulip/eeprom.c
	drivers/net/tulip/interrupt.c
	drivers/net/tulip/media.c
	drivers/net/tulip/pnic.c
	drivers/net/tulip/pnic2.c
	drivers/net/tulip/timer.c
	drivers/net/tulip/tulip.h
	drivers/net/tulip/tulip_core.c
	drivers/parisc/sba_iommu.c
	drivers/platform/x86/Kconfig
	drivers/scsi/megaraid/megaraid_mbox.c
	drivers/staging/cxt1e1/Kconfig
	drivers/usb/serial/digi_acceleport.c
	drivers/video/igafb.c
	drivers/watchdog/smsc37b787_wdt.c
	fs/configfs/inode.c
	fs/configfs/item.c
	fs/locks.c
	fs/squashfs/Kconfig
	include/linux/io-mapping.h
	include/linux/isdn.h
	include/linux/platform_data/ntc_thermistor.h
	include/media/videobuf-dma-sg.h
	include/target/configfs_macros.h
	net/netfilter/Kconfig
	sound/oss/Kconfig
	tools/perf/util/config.c

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 Documentation/power/basic-pm-debugging.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/power/basic-pm-debugging.txt b/Documentation/power/basic-pm-debugging.txt
index ddd7817..05a7fe7 100644
--- a/Documentation/power/basic-pm-debugging.txt
+++ b/Documentation/power/basic-pm-debugging.txt
@@ -173,7 +173,7 @@ kernel messages using the serial console.  This may provide you with some
 information about the reasons of the suspend (resume) failure.  Alternatively,
 it may be possible to use a FireWire port for debugging with firescope
 (ftp://ftp.firstfloor.org/pub/ak/firescope/).  On x86 it is also possible to
-use the PM_TRACE mechanism documented in Documentation/s2ram.txt .
+use the PM_TRACE mechanism documented in Documentation/power/s2ram.txt .
 
 2. Testing suspend to RAM (STR)
 
-- 
1.7.10.1.362.g242cab3

